.github/workflows/docker.yml

```diff
@@ -0,0 +1,75 @@
+name: Build & Push Docker Images
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+    - develop
+
+jobs:
+  pre_job:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v3
+        with:
+            username: ${{ secrets.DOCKERHUB_USERNAME }}
+            password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: List built images
+        run: docker images
+
+      - name: Build and Push Docker Image (Without Solvers)
```

Comment:
this one **also** has the `latest` tag

Reply:
the earlier one (with all solvers) should have had the `all` tag. I do have a local branch which fixes things up a bit for this workflow (including logging in and uploading these images in parallel jobs) and will write a PR soon, I might look into adding the multi-platform images there itself.